bench/wtperf/runners/parallel-pop-btree.wtperf

```diff
@@ -3,7 +3,7 @@
 conn_config="cache_size=200MB"
 table_config="type=file"
 transaction_config="isolation=snapshot"
-icount=10000000
+icount=500000000
 report_interval=5
 populate_ops_per_txn=100
-populate_threads=5
+populate_threads=24
```

Comment:
Please add a new test, rather than modifying the existing one? I suspect that will require an amendment to an evergreen.yml file as well to start running the test.

Reply:
Thanks. Good idea.